portal_tables/unify_grant_tables.py

```diff
@@ -0,0 +1,145 @@
+"""Combine Synapse projects that are separated 
+by grant into the CCKP admin project table.
+
+This script assumes the folders for PublicationView,
+DatasetView, and ToolView, respectively, remain 
+consistent in the CCKP admin project table.
+
+author: victor.baham
+"""
+
+# command line arguments: 1. Portal - Grants Merged Synapse ID 2. CCKP Admin Project Synapse ID
+# test value for argument 1: syn42801895
+# test value for argument 2: syn35629947 
+
+import argparse
+import synapseclient
+from synapseclient import Project, File, Folder
+from synapseclient import Schema, Column, Table, Row, RowSet, as_table_columns
+import itertools
+import pandas as pd
+
+def get_args():
+    #Retrieve Synapse table IDs from command line
+    parser = argparse.ArgumentParser(
+        description="Merge publications, datasets, and tools from Grants Merged table into CCKP admin table")
+    parser.add_argument("-p1", "--project1",
+                        type=str, required=True,
+                        help="Portal - Grants Merged Synapse ID.")
+    parser.add_argument("-p2", "--project2",
+                        type=str, required=True,
+                        help=("CCKP - MC2 Admin Synapse ID"))
+    parser.add_argument("--dryrun", action="store_true")
+    return parser.parse_args()
+
+def getManifestsFromGrant(syn,theParent):
```

Comment:
Try to be consistent with your naming format, e.g. above you used snake case for `get_args` and here you're using camel case for `getManifestsFromGrant`.  

If you want to adhere to [PEP 8](https://peps.python.org/pep-0008/#function-and-variable-names), then you should ideally use snake case for function (and variable) names.

Reply:
```suggestion
def getManifestsFromGrant(syn, theParent):
```

There should ideally be a space after the comma in the function parameters.